ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Debian.yml

```diff
@@ -116,6 +116,7 @@
   with_items: "{{ gcc_compiler }}"
   when:
     - (ansible_distribution_major_version != "9" and ansible_architecture != "armv7l")
+    - ansible_distribution_major_version != "10"
```

Comment:
This doesn't seem related to the PR title.
Also shouldn't we be saying which distribution this is? When we hit RHEL/Cent10 this conditional will trigger

Reply:
Nope, this is within the `Common/tasks/Debian.yml` file ; which only runs on Debian guests. However, I am going to remove this because I've done it a better way in my own testing, and it relates to a PR I'll be doing later on today.